includes/class-wcs-cart-renewal.php

```diff
@@ -156,13 +144,7 @@ public function setup_hooks() {
 	 * @since 1.0.0 - Migrated from WooCommerce Subscriptions v2.2.11
 	 */
 	public function attach_dependant_callbacks() {
-
-		if ( wcs_is_woocommerce_pre( '3.0' ) ) {
-			add_action( 'woocommerce_add_order_item_meta', array( &$this, 'add_order_item_meta' ), 10, 2 );
-			add_action( 'woocommerce_add_subscription_item_meta', array( &$this, 'add_order_item_meta' ), 10, 2 );
-		} else {
-			add_action( 'woocommerce_checkout_create_order_line_item', array( &$this, 'add_order_line_item_meta' ), 10, 3 );
-		}
+		add_action( 'woocommerce_checkout_create_order_line_item', array( &$this, 'add_order_line_item_meta' ), 10, 3 );
```

Comment:
Minor/optional, I realize this pre-dates your change, but we could drop the unnecessary `&`, since an object is always passed by reference:

```suggestion
		add_action( 'woocommerce_checkout_create_order_line_item', array( $this, 'add_order_line_item_meta' ), 10, 3 );
```

Reply:
Fixed with 844fcd004ef26af062d5a87fede9fa1e499af368